README.md

```diff
@@ -12,7 +12,7 @@ git clone git@github.com:morinlab/GAMBLR.git
 In Rstudio (on a gphost), set your working directory to the place you just cloned the repo.
 
 ```
-setwd("~/GAMBLR")
+setwd("~/GAMBLR-master")
```

Comment:
is this something new or have you made a new copy and this is an accidental push? Just confirming the status of this change

Reply:
This was updated to be directly compatible with the previous code chunk:
```
git clone git@github.com:morinlab/GAMBLR.git
```
I think this change can make its way to master, but the other path should be updated to be a placeholder, as discussed.